tests/test_operations.rs

```diff
@@ -49,12 +49,12 @@ fn test_op_log() {
     let stdout = test_env.jj_cmd_success(&repo_path, &["op", "log"]);
     let regex = Regex::new(r"\d\d years").unwrap();
     insta::assert_snapshot!(regex.replace_all(&stdout, "NN years"), @r###"
-    @  45108169c0f8 test-username@host.example.com NN years ago, lasted less than a microsecond
```

Comment:
I think we should leave one test for the relative timestamps in the oplog.

Reply:
Oops, definitely, good catch. I moved that to `test_op_log_template()`.